Q_R-AXIS_RCS_AUTOPILOT.s

```diff
@@ -487,7 +487,7 @@ RERRCALC	CAE	DAPTEMP1	# R-ERROR CALCULATION:
 		CAE	DAPTEMP2	# SECOND TERM CALCULATION:
 		EXTEND			# CDU ANGLE -ANGLE DESIRED (Z-AXIS)
 		MP	M32		# (CDUZ-CDUZD)*M32 SCALED AT PI RADIANS
-		AD	DELREROR	# KALCMANU INERFACE ERROR
+		AD	DELREROR	# KALCMANU INTERFACE ERROR
```

Comment:
This is `INERFACE` in the scan of [page 1451](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1451.jpg).